templates/cf.yml

```diff
@@ -618,6 +639,8 @@ properties:
     tls:
       ca_cert: ~
     etcd:
+      ca_cert: (( .properties.etcd.ca_cert ))
+      require_ssl: (( .properties.etcd.require_ssl ))
```

Comment:
The cluster configuration is directly under the etcd properties [line 582](https://github.com/cloudfoundry/cf-release/pull/978/files#diff-b9982126ee4438e20576a100670694d8R582). These properties are to provide our components (Doppler, TrafficController, MetronAgent, SyslogDrainBinder) with the etcd `ca_cert`, `require_ssl`, and `machines`